src/betterproto/__init__.py

```diff
@@ -688,6 +688,10 @@ def __repr__(self) -> str:
         ]
         return f"{self.__class__.__name__}({', '.join(parts)})"
 
+    def __rich_repr__(self) -> Iterable[Tuple[str, Any, Any]]:
+        for field_name in self._betterproto.sorted_field_names:
+            yield field_name, self.__raw_get(field_name), PLACEHOLDER
```

Comment:
I think this could be a `yield from`, something like this:
```suggestion
        yield (name, self.__raw_get(name), PLACEHOLDER for name in self._betterproto.sorted_field_names)
```

Reply:
Using `for ... yield` seems clearer, though.